containers/build.sh

```diff
@@ -22,8 +21,8 @@ if [[ -n $GITHUB_REF_NAME ]]; then
   fi
   sanitized=$(echo "$GITHUB_REF_NAME" | sed 's/[^a-zA-Z0-9.-]\+/-/g')
   OPEN_DEVIN_BUILD_VERSION=$sanitized
-  cache_tag+="-${sanitized}"
-  tags+=("$sanitized")
+  tag=${sanitized,,} # lower case is required in tagging
+  tags+=("$tag")
```

Comment:
```suggestion
  tags+=("$tag")
```
```suggestion
  tags+=("${sanitized,,}")
```

Reply:
Thanks for the suggestion. I actually intentionally split it into two lines because it's obscure and need an inline comment to improve readability.

Reply:
updated

Reply:
This syntax is bash-specific so I rewrote it.